src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -132,15 +133,15 @@ const BioAndDMSetting = ({
 };
 
 const styles = StyleSheet.create({
-  container: {
-    backgroundColor: colors.darkBlue,
+  container: (bgColor) => ({
+    backgroundColor: bgColor || COLORS.signed_primary,
     borderRadius: 15,
     paddingHorizontal: 12,
     marginTop: 20
-  },
-  editPromptLabel: {color: colors.blueSea10, textDecorationLine: 'underline'},
+  }),
+  editPromptLabel: {color: COLORS.blueSea, textDecorationLine: 'underline'},
   bioText: {
-    color: '#F5F5F5',
+    color: COLORS.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles have been updated to use the new colors from the `COLORS` object. The `container` style now takes a `bgColor` parameter that defaults to `COLORS.signed_primary` if not provided. The `editPromptLabel` and `bioText` styles have also been updated to use new color values.
```diff
- container: {
-   backgroundColor: colors.darkBlue,
- },
- editPromptLabel: {color: colors.blueSea10, textDecorationLine: 'underline'},
- bioText: {
-   color: '#F5F5F5',
+ container: (bgColor) => ({
+   backgroundColor: bgColor || COLORS.signed_primary,
+ }),
+ editPromptLabel: {color: COLORS.blueSea, textDecorationLine: 'underline'},
+ bioText: {
+   color: COLORS.lightgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->